worker/image-cache.js

```diff
@@ -11,39 +11,32 @@ imageToBase64Url = function(image) {
   return canvas.toDataURL("image/png");
 };
 
-imageFetch = function(success) {
-  var imageCategories = ["buildings", "nature", "objects", "people", "technology"];
+imageFetch = function(resolutionIndex, photoID, success) {
+  resolutionIndex = parseInt(resolutionIndex) || 0;
   var image = new Image();
-  var randomCategory = imageCategories[Math.floor(Math.random() * imageCategories.length)];
-  var baseUrl = 'https://source.unsplash.com/category/'
-  image.src =  baseUrl + randomCategory + '/2880x1800';
+  var imageResolution = ['1920x1080', '1920x1090', '1920x1100', '1920x1110', '1920x1120']
+  var baseUrl = 'https://source.unsplash.com/collection/220267/'
+  if (photoID) {
+    baseUrl = 'https://source.unsplash.com/' + photoID + '/'
+  }
+  image.src =  baseUrl + imageResolution[resolutionIndex];
   image.crossOrigin = 'Anonymous';
   image.onload = function() {
     success(image);
   };
 };
 
-updateImage = function(key) {
+updateImage = function(key, resolutionIndex, photoID) {
```

Comment:
Yeah, I cupped it n. I initially coded this whole new thing with static images for the initial set of 5. But now that's totally not required. Will remove photoID everywhere.